spec/delayed_paperclip/instance_methods_spec.rb

```diff
@@ -23,6 +23,7 @@
     it "clears instance variables" do
       dummy.instance_variable_set(:@_enqued_for_processing, ['foo'])
       dummy.instance_variable_set(:@_enqued_for_processing_with_processing, ['image'])
+      dummy.expects(:enqueue_post_processing_for).with('foo')
```

Comment:
I'm not sure what all this tests are doing, but without this line it was throwing an error - maybe someone can investigate it.